internal/pkg/cli/svc_override.go

```diff
@@ -18,21 +18,22 @@ import (
 	"github.com/spf13/cobra"
 )
 
-type overrideSvcVars struct {
+type overrideWorkloadVars struct {
 	overrideVars
 	envName string // Optional.
 }
 
-type overrideSvcOpts struct {
+type overrideWorkloadOpts struct {
```

Comment:
interestinggg I wonder if `svc/job deploy` can benefit from this pattern as well